tests/test_router.py

```diff
@@ -56,9 +57,9 @@ def test_add_route(self):
         assert len(router) == 0
 
         @add_route("/")
-        def x():
+        def random_method():  # pylint: disable=unused-variable
```

Comment:
Unused variable? That is a method, so I am not sure if `pylint` points it 🤔

Reply:
weird... but removed.